main.go

```diff
@@ -19,14 +18,20 @@ const MaxPeers = 10
 const MsgSize = 4096
 const IpString = "127.0.0.1"
 
-var logger *log.Logger = log.New(os.Stdout, "Mesh: ", log.Flags())
-
 func main() {
 	rand.Seed(time.Now().Unix())
+	// logger configuration
+	cfg := zap.NewProductionConfig()
+	cfg.OutputPaths = []string{"/tmp/agents.log"}
```

Comment:
I think file name should contain some timestamp or similar in order to create another (unique) log file for each run.

After we add commands it will be possible to specify output file from the command line flags and if the flag is not set, file should be generated as I mentioned above